dask/array/core.py

```diff
@@ -27,6 +27,7 @@
 from typing import Any, TypeVar, Union, cast
 
 import numpy as np
+import pandas as pd
```

Comment:
This import is causing our `imports` builds to fail as `pandas` isn't a requirement for using `dask.array`